pinot-spi/src/main/java/org/apache/pinot/spi/ingestion/batch/BatchConfig.java

```diff
@@ -185,6 +189,14 @@ public boolean isExcludeSequenceId() {
     return _excludeSequenceId;
   }
 
+  public String getPartitionId() {
+    return _partitionId;
+  }
+
+  public long getSegmentUploadTimeMs() {
```

Comment:
nit: getter order should match declaration order: _partitionId > sequenceId > segmentUploadTimeMs